### PR TITLE
🌱 cron: Add 14k GitLab projects

### PR DESCRIPTION
**Pull request by raghavkaul**
#### What kind of change does this PR introduce?
Adds 14k GitLab projects, for a total of ~15k. Ran `make validate-projects`.

### CODE REVIEW

File: cron/internal/data/gitlab-projects.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-repo,metadata
+repo,metadat,
```
Comment (spencerschrock) [inline cron/internal/data/gitlab-projects.csv:1]: typo/unintended change

**Review by spencerschrock — COMMENTED** [review]
Can you manually run `make add-projects` against this file? Or can you fix the makefile target?

Neither add or validate projects is looking at this file.

File: cron/internal/data/gitlab-projects.csv [hunk 2]
```diff
@@ -5499,4 +5499,14479 @@ https://gitlab.com/silenteer-oss/tutum/titan,
 https://gitlab.com/lthn.io/projects/chain/miner,
 https://gitlab.com/bz1/aiida-castep,
 https://gitlab.com/fortitudetec/elucidation,
-https://gitlab.com/mschlueter/laravel-force-https,
\ No newline at end of file
+https://gitlab.com/mschlueter/laravel-force-https,
+https://gitlab.com/highdiceroller/hdrolle,
+https://gitlab.com/jbtcd/threesixtyj,
```
Comment (spencerschrock) [inline cron/internal/data/gitlab-projects.csv:5504]: Thanks for fixing validate-projects. Do we want to do the same with `add-projects` so the list is alphabetical?

**Review by spencerschrock — APPROVED** [review]

File: cron/internal/data/gitlab-projects.csv (hunk 2 above)
Comment (raghavkaul) [reply cron/internal/data/gitlab-projects.csv:5504]: Hm, there would be two additional csvs, and one of them may not always have changed... so seems excessive to me.

**Pull request merged**